### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by MarkVed17**
~ Added Typography page

### CODE REVIEW

**Review by horsemaker — APPROVED** [review]
LGTM! 👍

**Review by msdeshmukh009 — APPROVED** [review]
The code looks fine but not sure about the file structure as I think the typography.css should contain all the typography related styles.

File: components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  margin-top: 1rem;
+  color: var(--font-color);
```
Comment (vijaytembugade) [inline components/typography/typography.css:7]: You can create one global class of container.

File: components/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+
+.headings-format {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (vijaytembugade) [inline components/typography/typography.css:26]: Same here, can create Flex-container-verticle class, which has display:flex, flex-direction:column, and align-item:center, which can used anywhere, not need define everytime.

**Review by vijaytembugade — APPROVED** [review]

File: components/typography/typography.css (hunk 2 above)
Comment (MarkVed17) [reply components/typography/typography.css:26]: Noted.

**Pull request merged**